README.md

```
@@ -104,30 +98,18 @@ SOFTWARE.
-[i386-shield]: https://img.shields.io/badge/i386-no-red.svg
-[issue]: https://github.com/hassio-addons/addon-grafana/issues
-[license-shield]: https://img.shields.io/github/license/hassio-addons/addon-grafana.svg
+[github-actions-shield]: https://github.com/ysm-hassio/addon-grafana/workflows/CI/badge.svg
+[github-actions]: https://github.com/ysm-hassio/addon-grafana/actions
+[issue]: https://github.com/ysm-hassio/addon-grafana/issues
+[license-shield]: https://img.shields.io/github/license/ysm-hassio/addon-grafana.svg
 [maintenance-shield]: https://img.shields.io/maintenance/yes/2025.svg
-[patreon-shield]: https://frenck.dev/wp-content/uploads/2019/12/patreon.png
-[patreon]: https://www.patreon.com/frenck
 [project-stage-shield]: https://img.shields.io/badge/project%20stage-production%20ready-brightgreen.svg
 [reddit]: https://reddit.com/r/homeassistant
```

markdownlint-cli2 (0.18.1)

Link and image reference definitions should be needed
Unused link or image reference definition: "reddit"

(MD053, link-image-reference-definitions)